projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -276,12 +276,9 @@ export function JetpackAIInterstitial() {
 	const { detail } = useProduct( slug );
 	const { onClickGoBack } = useGoBack( { slug } );
 
-	const currentTier = detail?.[ 'ai-assistant-feature' ]?.[ 'current-tier' ]?.value;
 	const nextTier = detail?.[ 'ai-assistant-feature' ]?.[ 'next-tier' ]?.value;
```

Comment:
I think checking if `next-tier` has any value is enough. But if you really want to get a value from it, I'd suggest using the `limit`, as that will never be `0` so you can rely on `! nextTier`.

Besides, a non existent property on an object will give you `undefined`, not `null` (might also suggest using strict comparison on line 281)

Reply:
He may want to get a value from it to set as quantity on the interstitial. This value will be used to determine the `q` parameter on the checkout URL, so it's necessary. Using `value` or `limit` for the `q` parameter will have no difference, unless we start changing the limits of a given tier without changing it's quantity (unlikely).

+1 regarding the strict comparison, or maybe something like:

```
const nextTier = detail?.[ 'ai-assistant-feature' ]?.[ 'next-tier' ] || null;
```

Then

```
if ( !! nextTier ) {
```

And also:

```
quantity={ nextTier.value }
```